src/discord/commands/LowestbinCommand.js

```diff
@@ -0,0 +1,74 @@
+/*
+ CREDIT: Idea by Aura
+ Discord: Aura#5051
+ Minecraft username: _aura
+*/
+const {SlashCommandBuilder} = require('@discordjs/builders')
+const fetch = require("axios");
+const {Index} = require("flexsearch");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('lowestbin')
+        .setDescription('Display basic info about the client.')
+        .addStringOption(option =>
+            option.setName('item')
+                .setDescription('Item you wanna search the lowest bin of.')
```

Comment:
avoid using "you" and "wanna"
```suggestion
                .setDescription('Item name')
```